CHANGELOG

```diff
@@ -1,6 +1,11 @@
 v3.3.8 (XXXX-XX-XX)
 -------------------
 
+* fixed issue #3811: gharial api is now checking existence of _from and _to vertices
+  during edge creation
+
+* fixed issue #4934: Wrong used GeoIndex depending on FILTER order
```

Comment:
This seems to not be within this pr.
Can we check what the status of this issue is and keep/remove it from here?

Reply:
Please move to v3.3.9 as we release 3.3.8 without it.